util.go

```diff
@@ -66,3 +70,13 @@ func sliceElemType(v reflect.Value) reflect.Type {
 	}
 	return indirectType(elemType)
 }
+
+// appendComment adds comment in the header of the query into buffer
+func appendComment(b []byte, name string) []byte {
```

Comment:
Here need to filter 0x00

`appendComment(nil, string([]byte{'*', 0, 0, 0, 0, 0, '/'}))`

Reply:
Thank you. Added fix + test